app/livechat/client/views/app/tabbar/chatRoomSearchItem.html

```diff
@@ -0,0 +1,18 @@
+<template name='chatRoomSearchItem'>
```

Comment:
You don't need this template, you can use the `chatRoomHistoryItem`.
Please, remove it.

Reply:
i can use the chatRoomHistory but it may get complicated first the styling is diff then i have to create a helper method to check image path of the user and to another to check messages count and one to check time with string

Reply:
I have explained before:
- When you're searching or listing in the `customerChatRoomHistory` template, you'll display the results using the `customerChatRoomHistoryItem` template;
When you're searching or listing in the `customerChatHistoryMessages` template you'll display the results using the `customerChatHistoryMessagesItem` template;